.eslintrc.js

```diff
@@ -25,6 +26,7 @@ module.exports = {
   },
   // add your custom rules here
   rules: {
+    '@typescript-eslint/ban-ts-comment': ['warn'],
```

Comment:
I think this will be helpful for encouraging devs to justify cases of ts-ignore throughout the codebase.